rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -166,7 +163,9 @@ class DIETClassifier(EntityExtractor):
         # dropout rate for rnn
         DROPRATE: 0.2,
         # use a unidirectional or bidirectional encoder
-        UNIDIRECTIONAL_ENCODER: False,
+        UNIDIRECTIONAL_ENCODER: True,
```

Comment:
```suggestion
        UNIDIRECTIONAL_ENCODER: False,
```
let's keep it False for Diet